src/MemStore.cc

```diff
@@ -25,7 +25,7 @@
 #include "tools.h"
 
 /// shared memory segment path to use for MemStore maps
-static const SBuf MapLabel("cache_mem_map");
+static const char *MapLabel = "cache_mem_map";
```

Comment:
We could improve const-correctness and apply AAA to this modified-for-other-reasons code, but it will make its formatting inconsistent with the two variable declarations that follow immediately below. Your call.

```suggestion
static const auto MapLabel = "cache_mem_map";
```

Reply:
Let's move forward with the changes, it won't hurt